packages/sdk/src/execute/execute.utils.ts

```diff
@@ -121,8 +120,8 @@ const batchExecuteWithNearAccount = async (
           contractId: call.contractAddress,
           methodName: call.methodName,
           args: call.args,
-          gas: new BN(call.gas),
-          attachedDeposit: new BN(call.deposit),
+          gas: BigInt(call.gas.toString()),
```

Comment:
Shouldn't `call.gas` be a string or Bigint already?

Reply:
Also, shouldn't this be totally fine with taking a string instead of casting to BigInt again? Pretty sure it's sent to RPC as a string.